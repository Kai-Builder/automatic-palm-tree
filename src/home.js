import Charts from './charts';
import Hello from './hello';
import Links from './links';
import React from 'react';
import {home} from './home.scss';

const Home = () => (
  <div className={home}>
    <Hello />
    <Links />
    <Charts />
  </div>
);

export default Home;
