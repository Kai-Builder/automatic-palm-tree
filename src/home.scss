.home {
  border: 1px solid blue;
  display: grid;

  grid-template-areas:
    'hello'
    'links'
    'main';
 }
