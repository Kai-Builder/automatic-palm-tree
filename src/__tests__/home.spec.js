import {Home} from '../home';
import React from 'react';
import {shallow} from 'enzyme';

describe('Home', () => {
  const requiredProps = () => ({
  });
  const render = (props = requiredProps()) => shallow(<Home {...props}/>);

  test('component renders', () => {
    const wrapper = render();

    expect(wrapper).toMatchSnapshot();
  });
});
