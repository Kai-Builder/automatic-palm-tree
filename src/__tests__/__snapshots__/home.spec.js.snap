// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders 1`] = `
<section
  className=""
>
  <Hello
    className=""
  />
  <SimpleAreaChart />
</section>
`;
